tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
+{"root":["./src/app.ts","./src/server.ts","./src/controllers/bookcontroller.ts","./src/controllers/commentcontroller.ts","./src/controllers/discussioncontroller.ts","./src/controllers/maincontroller.ts","./src/controllers/usercontroller.ts","./src/db/connect.ts","./src/errors/bad-request.ts","./src/errors/custom-api.ts","./src/errors/index.ts","./src/errors/not-found.ts","./src/errors/unauthenticated.ts","./src/middleware/authentication.ts","./src/middleware/error-handler.ts","./src/models/book.ts","./src/models/comment.ts","./src/models/discussion.ts","./src/models/user.ts","./src/routes/bookroutes.ts","./src/routes/commentroutes.ts","./src/routes/discussionroutes.ts","./src/routes/mainrouter.ts","./src/routes/userroutes.ts","./src/services/mailjetservice.ts","./src/types/express.d.ts","./src/utils/swagger.ts"],"version":"5.6.3"}
```

Comment:
what does this do?

Reply:
I deleted it. I don’t know why or when it was added. Maybe it was added automatically.